website/src/examples/collision-filter-extension.js

```diff
@@ -0,0 +1,126 @@
+import React, {Component} from 'react';
+import {readableInteger} from '../utils/format-utils';
+import {MAPBOX_STYLES, GITHUB_TREE} from '../constants/defaults';
+import App, {COLORS_STEPS} from 'website-examples/collision-filter/app';
+
+import {makeExample} from '../components';
+
+class CollisionDemo extends Component {
+  static title = 'California Collision Test';
+
+  static code = `${GITHUB_TREE}/examples/website/collision-filter`;
+
+  static parameters = {
+    routeName: {
```

Comment:
I don't think this filter adds much value. Just show everything.

Reply:
wip